examples/model_service_template/mxnet_vision_batching.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think it's better upgrade current mxnet_vision_service.py to support batching. so we don't need maintain two set of model archive and template in future.

Reply:
Shall do

Reply:
I am planning to remove all the template files and just have this one for vision. I will do this separately.